bgpd/bgp_zebra.c

```diff
@@ -750,6 +750,14 @@ bool bgp_zebra_nexthop_set(union sockunion *local, union sockunion *remote,
 								? peer->conf_if
 								: peer->ifname,
 							peer->bgp->vrf_id);
+			else if (peer->update_if)
```

Comment:
Is this a valid case for LL peering?

Reply:
as per Donald remark, this should be a valid case, only if update-source command is used.